test/built-ins/Promise/anySettled/S25.4.4.3_A2.2_T3.js

```diff
@@ -3,14 +3,15 @@
 
 /*---
 info: |
-    Promise.race rejects when GetIterator() returns an abrupt completion
+    Promise.anySettled rejects when GetIterator() returns an abrupt completion
     4. Let iterator be GetIterator(iterable).
     5. IfAbruptRejectPromise(iterator, promiseCapability)
-es6id: S25.4.4.3_A2.2_T3
+esid: sec-promise-anysettled
 author: Sam Mikes
-description: Promise.race rejects if GetIterator throws
+description: Promise.anySettled rejects if GetIterator throws
 features: [Symbol.iterator]
 flags: [async]
+features: [Promise.anySettled]
```

Comment:
another minor typo: duplicate features.

```suggestion
features: [Symbol.iterator, Promise.anySettled]
flags: [async]
```